<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>722</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <property name="spacing">
     <number>8</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBoxSection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rBtnFinance">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Finance</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnTask">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Task</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnInventory">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Inventory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnPartner">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Partner</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnSale">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Sale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnPurchase">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Purchase</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionCheckUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionInsertNode"/>
    <addaction name="actionAppendNode"/>
    <addaction name="actionNewBranch"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="actionClearColor"/>
    <addaction name="separator"/>
    <addaction name="actionAppendEntry"/>
    <addaction name="actionJumpEntry"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionTags"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionSignIn"/>
    <addaction name="actionSignOut"/>
    <addaction name="separator"/>
    <addaction name="actionReconnect"/>
    <addaction name="separator"/>
    <addaction name="actionEmail"/>
    <addaction name="actionWorkspace"/>
    <addaction name="actionExpireDate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAccount"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInsertNode"/>
   <addaction name="actionAppendNode"/>
   <addaction name="actionNewBranch"/>
   <addaction name="separator"/>
   <addaction name="actionRename"/>
   <addaction name="actionClearColor"/>
   <addaction name="separator"/>
   <addaction name="actionAppendEntry"/>
   <addaction name="actionJumpEntry"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionMarkAll"/>
   <addaction name="actionMarkToggle"/>
   <addaction name="actionMarkNone"/>
   <addaction name="separator"/>
   <addaction name="actionStatement"/>
   <addaction name="actionSettlement"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionInsertNode">
   <property name="text">
    <string>Insert Node...</string>
   </property>
   <property name="toolTip">
    <string>Insert a new node at the end of the current level</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete...</string>
   </property>
   <property name="toolTip">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionAppendNode">
   <property name="text">
    <string>Append Node...</string>
   </property>
   <property name="toolTip">
    <string>Append a new node under the selected branch</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionJumpEntry">
   <property name="text">
    <string>Jump Entry</string>
   </property>
   <property name="toolTip">
    <string>Jump to the linked entry in the other node tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename...</string>
   </property>
   <property name="toolTip">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About YTX</string>
   </property>
   <property name="toolTip">
    <string>About YTX</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutRole</enum>
   </property>
  </action>
  <action name="actionMarkAll">
   <property name="text">
    <string>Mark All</string>
   </property>
   <property name="toolTip">
    <string>Mark All</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionMarkNone">
   <property name="text">
    <string>Mark None</string>
   </property>
   <property name="toolTip">
    <string>Mark None</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionMarkToggle">
   <property name="text">
    <string>Mark Toggle</string>
   </property>
   <property name="toolTip">
    <string>Mark Toggle</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAppendEntry">
   <property name="text">
    <string>Append Entry</string>
   </property>
   <property name="toolTip">
    <string>Append Entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExportExcel">
   <property name="text">
    <string>Export Excel...</string>
   </property>
   <property name="toolTip">
    <string>Export Excel</string>
   </property>
  </action>
  <action name="actionStatement">
   <property name="text">
    <string>Statement</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSettlement">
   <property name="text">
    <string>Settlement</string>
   </property>
   <property name="toolTip">
    <string>Settlement</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSignIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sign In...</string>
   </property>
   <property name="toolTip">
    <string>Sign In</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSignOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
   <property name="toolTip">
    <string>Sign Out</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionEmail">
   <property name="text">
    <string>Email</string>
   </property>
   <property name="toolTip">
    <string>Email</string>
   </property>
  </action>
  <action name="actionWorkspace">
   <property name="text">
    <string>Workspace</string>
   </property>
   <property name="toolTip">
    <string>Workspace</string>
   </property>
  </action>
  <action name="actionClearColor">
   <property name="text">
    <string>Clear Color</string>
   </property>
   <property name="toolTip">
    <string>Clear Color</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reconnect</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="actionNewBranch">
   <property name="text">
    <string>New Branch...</string>
   </property>
   <property name="toolTip">
    <string>Create a new branch node to organize order nodes</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExpireDate">
   <property name="text">
    <string>Expire Date</string>
   </property>
  </action>
  <action name="actionCheckUpdates">
   <property name="text">
    <string>Check Updates</string>
   </property>
   <property name="toolTip">
    <string>Check Updates</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionTags">
   <property name="text">
    <string>Tags...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
